content/22.vaccines-trad.md

```diff
@@ -295,16 +305,17 @@ Protein subunit vaccine development efforts for both SARS-CoV-1 and MERS-CoV usu
+Other strategies investigating the potential use of the full length S DNA have also been investigated in mice and rhesus macaques, which elicited immune responses [@doi:10.1038/ncomms8712], but these responses were not as effective as the combination of S DNA and the S1 subunit protein together [@doi:10.1038/ncomms8712; @doi:10.1016/j.antiviral.2016.07.015].
 No protein subunit vaccine for MERS-CoV has progressed beyond preclinical research to date.
 VLPs have been investigated for development of vaccines against MERS and SARS [@doi:10.1016/j.vaccine.2020.07.003; @doi:10.1016/j.vaccine.2014.04.016] including testing in animal models [@doi:10.18632/oncotarget.8475; @doi:10.1007/s12250-018-0064-8], but once again, only preclinical data against HCoV has been collected [@doi:10.1186/s12929-020-00695-2].
-However, protein subunit vaccines do play a role in public health and have contributed to vaccination against hepatitis B [@doi:10/d4g86c] and pertussis [@doi:10.1128/mBio.01339-14; @doi:10.1371/journal.ppat.1003418] since the 1980s and will likely continue to contribute to public health for the foreseeable future due to ongoing research in vaccines against influenza, SARS-CoV-2, Epstein-Barr virus, dengue virus, and human papillomavirus among others [@doi:10.1016/j.micinf.2014.12.006; @url:https://covid19.trackvaccines.org/vaccines; @doi:10.1080/21655979.2016.1191707].
+However, protein subunit vaccines do play a role in public health and have contributed to vaccination against hepatitis B [@doi:10/d4g86c] and pertussis [@doi:10.1128/mBio.01339-14; @doi:10.1371/journal.ppat.1003418] since the 1980s and human papillomavirus since 2006 [@doi:10.1188/06.CJON.559-560].
+They are likely to continue to contribute to public health for the foreseeable future due to ongoing research in vaccines against influenza, SARS-CoV-2, Epstein-Barr virus, dengue virus, and human papillomavirus among others [@doi:10.1016/j.micinf.2014.12.006; @url:https://covid19.trackvaccines.org/vaccines; @doi:10.1080/21655979.2016.1191707].
 
 #### Application to COVID-19
 
-The development of subunit vaccines against SARS-CoV-2 is a remarkable achievement given the short period of time since the emergence of SARS-CoV-2 in late 2019, particularly considering these types of vaccines have not played a major role in previous pandemics.
+The development of subunit vaccines against SARS-CoV-2 is a remarkable achievement given the short period of time since the emergence of SARS-CoV-2 in late 2019, particularly considering these types of vaccines have not played a major role in previous pandemics comapred to LAV and IWV vaccines.
```

Comment:
```suggestion
The development of subunit vaccines against SARS-CoV-2 is a remarkable achievement given the short period of time since the emergence of SARS-CoV-2 in late 2019, particularly considering these types of vaccines have not played a major role in previous pandemics compared to LAV and IWV vaccines.
```